Exiled.API/Enums/CameraType.cs

```diff
@@ -44,12 +47,15 @@ public enum CameraType
         #endregion
 
         #region Hcz
+        [Obsolete("Has been removed from the game")]
         Hcz049Armory,
         Hcz049ContChamber,
         Hcz049ElevTop,
         Hcz049Hallway,
         Hcz049TopFloor,
+        [Obsolete("Has been renamed Hcz049Outside")]
         Hcz049Tunnel,
+        Hcz049Outside = Hcz049Tunnel,
```

Comment:
why exactly are you using an obsolete enum item right after the line that makes it obsolete lol

Reply:
because they rename the name of the Camera so Hcz049Tunnel will be removed in futur and being replace by Hcz049Outside

Reply:
ok I got that, but like. make the obsolete item use the non-obsolete enum, not vice versa